daal4py/sklearn/_utils.py

```diff
@@ -17,26 +17,21 @@
 
 import numpy as np
 
-from daal4py import __daal_link_version__, __daal_run_version__
+from daal4py import __daal_link_version__ as dv, __has_dist__
 
-def daal_check_version(trad_target, dpcpp_target = None):
-    daal_run_version = tuple(map(int, (__daal_run_version__[0:4], __daal_run_version__[4:8])))
-    daal_link_version = tuple(map(int, (__daal_link_version__[0:4], __daal_link_version__[4:8])))
-    result = True
-    if daal_run_version[1] > 100:
-        # daal dpcpp
-        result = result and daal_run_version >= dpcpp_target if dpcpp_target is not None else False
+def daal_check_version(rule):
```

Comment:
If argument will be *argv, than you may not need to do this:
```python
 if daal_check_version(((2020, 2), (2021, 107, 'B'))):
```
And you can write like this:
```python
 if daal_check_version((2020, 2), (2021, 107, 'B')):
```

Reply:
currently it's a way how to identify format - single product version or multiple.  Would prefer to add brackets instead of writing new logic for formats